apis/datadoghq/v1alpha1/secret.go

```diff
@@ -26,17 +25,21 @@ func getKeysFromCredentials(credentials *datadoghqv1alpha1.DatadogCredentials) m
 	return data
 }
 
-// For each of the API key and app key, check if:
+// CheckAPIKeySufficiency use to check for the API key if:
 // 1. an existing secret is defined, or
 // 2. the corresponding env var is defined (whether in ENC format or not)
 // If either of these is true, the secret is not needed for that particular key.
```

Comment:
```suggestion
// If either of these is true, the secret is not needed.
```

(this comment originally was meant to apply to both functions)